Bento/Views/BentoReusableView.swift

```diff
@@ -1,20 +1,23 @@
-protocol BentoReusableView: NativeView {
+public protocol ViewStorageOwner: NativeView {
+    var storage: [StorageKey: Any] { get set }
+}
+
+protocol BentoReusableView: ViewStorageOwner {
     var containedView: NativeView? { get set }
     var contentView: NativeView { get }
     var component: AnyRenderable? { get set }
-    var storage: [StorageKey: Any] { get set }
 }
 
-struct StorageKey: Hashable {
+public struct StorageKey: Hashable {
```

Comment:
Yes, because there are composites in BabylonUI and they should call these lifecycle methods too (although they don’t).

Reply:
Then, do we need `ViewStorage.init` as `public` ?

Reply:
Noice catch.

Reply:
With 0b32de8 , this type doesn't need to be `public` for now until making `BentoReusableView` public.

But I guess it will become `public` again soon anyway, so I will ignore this.